integration-tests/scaffolding/helpers.ts

```diff
@@ -262,15 +265,89 @@ export async function setEpochLength(keys: KeyringPair, epochLength: number): Pr
   }
 }
 
-export async function createGraphChangeSchema(): Promise<u16> {
-  const [createSchemaEvent, eventMap] = await ExtrinsicHelper
-    .createSchema(devAccounts[0].keys, AVRO_GRAPH_CHANGE, "AvroBinary", "OnChain")
-    .fundAndSend();
-  assert.notEqual(eventMap["system.ExtrinsicSuccess"], undefined);
-  if (createSchemaEvent && ExtrinsicHelper.api.events.schemas.SchemaCreated.is(createSchemaEvent)) {
-    return createSchemaEvent.data.schemaId;
+export async function getOrCreateGraphChangeSchema(): Promise<u16> {
+  if (process.env.CHAIN_ENVIRONMENT === "rococo") {
+    const ROCOCO_GRAPH_CHANGE_SCHEMA_ID: u16 = new u16(ExtrinsicHelper.api.registry, 53);
+    return ROCOCO_GRAPH_CHANGE_SCHEMA_ID;
   } else {
-    assert.fail("failed to create a schema")
+    const [createSchemaEvent, eventMap] = await ExtrinsicHelper
+      .createSchema(devAccounts[0].keys, AVRO_GRAPH_CHANGE, "AvroBinary", "OnChain")
```

Comment:
Is it useful to use `getDefaultFundingSource()` here?

Reply:
It may be if it is decided to create schemas on testnet. Since the `getOrCreate` schema helper functions are currently only creating schemas when executing against a development node, I left this alone.

Reply:
I made a mistake in that comment:
The else block of the branch executes when `CHAIN_ENVIRONMENT` is anything but "rococo". Technically we should only reference those `devAccounts` when `CHAIN_ENVIRONMENT` is "local", but the only other `CHAIN_ENVIRONMENT` supported by the testing framework is "local" currently. 


There may be a case to add a "mainnet" environment in the future